src/NuGetForUnity/Editor/NugetPreferences.cs

```diff
@@ -72,6 +74,34 @@ public override void OnGUI(string searchContext)
                 NugetHelper.NugetConfigFile.Verbose = verbose;
             }
 
+            EditorGUILayout.BeginHorizontal();
+            {
+                var packagesConfigPath = NugetHelper.NugetConfigFile.RelativePackagesConfigPath;
+                EditorGUILayout.LabelField($"Packages Config path: {packagesConfigPath}");
+                packagesConfigPath = Path.GetFullPath(packagesConfigPath);
+                if (GUILayout.Button("Browse"))
+                {
+                    var newPath = EditorUtility.OpenFolderPanel("Select Folder", packagesConfigPath, "");
+
+                    if (!string.IsNullOrEmpty(newPath) && newPath != packagesConfigPath)
+                    {
+                        var pathCheck = NugetHelper.GetProjectRelativePath(newPath);
+
+                        // make sure the path is within Assets directory
+                        if (!pathCheck.StartsWith("Assets"))
```

Comment:
I think we can allow the file to be placed outside the "Assets" folder. I would only show a warning that this disables the functionality of restoring packages if the `package.config` file is changed "on the disk".